cellbase-app/app/cloud/kubernetes/charts/cellbase/templates/cert-issuer/private-issuer.yaml

```diff
@@ -0,0 +1,52 @@
+{{- if .Values.tls.privateIssuer.enabled }}
+##################
+## cert-manager resources
+##
+## self-signed-issuer -> self-signed-ca-cert -> ca-issuer
+##
+##################
+# self-signed issuer
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: {{ .Values.tls.privateIssuer.selfSignedName }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "opencga.labels" . | nindent 4 }}
+spec:
+  selfSigned: {}
+---
+# self-signed-ca-cert for the CA issuer
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: {{ .Values.tls.privateIssuer.caName }}-selfsigned-cert
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "opencga.labels" . | nindent 4 }}
```

Comment:
same here

Reply:
and in some other places. I won't mark them all

Reply:
fix it :)